app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessTypeController.scala

```diff
@@ -64,24 +63,15 @@ class BusinessTypeController @Inject()(
         .fold(
           formWithErrors => Ok(html.business_type(formWithErrors)),
           validatedBusinessType => {
-            if (validatedBusinessType == Invalid)
```

Comment:
would this lead to technical error if user manipulates the form and passes some random string to businessType and submits the form ?